src/test/java/uk/ac/ebi/eva/pipeline/io/writers/StatisticsMongoWriterTest.java

```diff
@@ -99,11 +103,12 @@ public void shouldCreateIndexesInCollection() throws Exception {
         // do the checks
         DBCollection statsCollection = mongoRule.getCollection(databaseName, jobOptions.getDbCollectionsStatsName());
 
-        // check vid has an index
+        // check there is an index in chr + start + ref + alt + sid + cid
         assertEquals("[{ \"v\" : 1 , \"key\" : { \"_id\" : 1} , \"name\" : \"_id_\" , \"ns\" : \"" + databaseName +
-                        ".populationStatistics\"}, { \"v\" : 1 , \"unique\" : true , \"key\" : { \"vid\" : 1 , \"sid\" : 1 , " +
-                        "\"cid\" : 1} , \"name\" : \"vscid\" , \"ns\" : \"" + databaseName + ".populationStatistics\"}]",
-                statsCollection.getIndexInfo().toString());
+                             ".populationStatistics\"}, { \"v\" : 1 , \"unique\" : true , \"key\" : " +
+                             "{ \"chr\" : 1 , \"start\" : 1 , \"ref\" : 1 , \"alt\" : 1 , \"sid\" : 1 , \"cid\" : 1} , " +
+                             "\"name\" : \"vscid\" , \"ns\" : \"" + databaseName + ".populationStatistics\"}]",
+                     statsCollection.getIndexInfo().toString());
```

Comment:
This test could compare the DBObjects extracted from `getIndexInfo()` using the `equals()` method.